spharpy/classes/sh.py

```diff
@@ -4,15 +4,14 @@
 import numpy as np
 import pyfar as pf
 import spharpy as sy
+from abc import ABC, abstractmethod
 
 
-class SphericalHarmonicDefinition:
-    """Class storing the definition of spherical harmonics.
+class _SphericalHarmonicBase(ABC):
+    """Base class related to spherical harmonics.
```

Comment:
Could you please add a short explanation here similar to the one in the PR ("Abstract base class for SphericalHarmonicDefinition for more flexible re-use in case n_max is not required or a getter only property"). This would help future developers to decide whether to inherit from  `_SphericalHarmonicBase` or `SphericalHarmonicDefinition`.